### DISCUSSION TITLE
How can I check the balance of a specific token at a specific address?

### DISCUSSION THREAD

**Discussion by 621625** [foundry-rs/foundry#1784] [Help]
Hello,

I'm MB. A very nice and polite guy.

How can I check the balance of a specific token at a specific address?

Let's say Dai token at 0x0xxxxxx (Polygon).

**Comment by 621625** [comment]
Looking for the easiest way, preferably a one-liner if there is one with forge or cast.

**Comment by onbjerg** [comment]
The book has an example like that: https://book.getfoundry.sh/reference/cast/cast-call.html#examples. Just substitute `0xC02..Cc2` with the token address, `0x...` for the address you want to check the balance of and remember to set `ETH_RPC_URL` :)
